endpointmanager/pkg/endpointmanager/healthitproduct.go

```diff
@@ -61,7 +63,14 @@ func (hitp *HealthITProduct) Equal(hitp2 *HealthITProduct) bool {
 	if hitp.APIURL != hitp2.APIURL {
 		return false
 	}
-	if !cmp.Equal(hitp.CertificationCriteria, hitp2.CertificationCriteria) {
+
+	// This Transformer sorts a []int.
+	trans := cmp.Transformer("Sort", func(in []int) []int {
```

Comment:
This seems to do the same thing as the `certificationCriteriaMatch` function in the `productquerier.go` file, but is simpler. Is there a reason why we couldn't do this in the `productquerier.go` file?

Reply:
We can do the same thing, I just updated the certificationCriteriaMatch to do the same thing!